src/Commands/Handler/Internal/ClientJoiningHandler.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace CSM.Commands.Handler.Internal
 {
-    class ClientJoiningHandler : CommandHandler<ClientJoiningCommand>
+    internal class ClientJoiningHandler : CommandHandler<ClientJoiningCommand>
```

Comment:
Is there a reason why this class is marked as internal?

Reply:
Ahh, missed that one. Should be public. Will fix this afternoon.